### PR TITLE
DOCS-9196 PR to move docs from working branch to prod.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/install-self-managed.adoc [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+|===
+
+== Step 2: Download the rtfctl Utility
+
+[IMPORTANT]
+====
+The tasks in the section must be performed by an IT administrator.
+====
+
+Runtime Fabric on Self-managed Kubernetes uses the `rtfctl` command-line utility for installation and management tasks. See xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool].
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/install-self-managed.adoc]: @sleslie23 there are a few new commands we have added to rtfctl that need to added to this href

Comment (kmendiratta-crm) [reply modules/ROOT/pages/install-self-managed.adoc]:
```
/rtfctl --help
rtfctl controls a Runtime Fabric cluster
	
Find more information at: https://anypoint.mulesoft.com/runtimefabric

Usage:
  rtfctl [command]

Available Commands:
  apply           Apply changes to an RTF configuration
  backup          Backup local Runtime Fabric state
  disk            Print the disk space used by an application
  get             Display one or many resources
  heapdump        Trigger a JVM heap dump
  help            Help about any command
  memory          Display the JVM memory usage in a pod
  package         Create a debugging information package for an application
  ping            Test network connectivity
  report          Generate diagnostics report
  restart         Restart an application
  restore         Restore local Runtime Fabric state from a backup
  status          Display status of the Runtime Fabric
  test            Test Runtime Fabric environment
  threaddump      Print the JVM thread dump for an application
  uninstall       Uninstall Runtime Fabric
  update          Update rtfctl
  upgrade-cluster Upgrade Runtime Fabric cluster
  validate        Validate cluster for Runtime Fabric installation
  version         Print the version information

```

Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Thanks a lot @kmendiratta-crm !!!

Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Fixed.

**Pull request merged**